WirelessTelemServer.py

```diff
@@ -7,49 +7,57 @@
 # Contains list of all connected users
 USERS = set()
 
+
 def get_server(ip, port):
     return websockets.serve(on_client_connect, ip, port)
 
+
 # Main server entry point
 async def on_client_connect(websocket, path):
-    
-    connection_addr = f'{websocket.remote_address[0]}:{websocket.remote_address[1]}'
-    print(f'Connected to client at {connection_addr} on {datetime.datetime.now()}.')
+    connection_addr = (f'{websocket.remote_address[0]}:' +
+                       f'{websocket.remote_address[1]}')
 
-    await websocket.send(json.dumps({'type':'connected','payload':'true', 'timestamp':time.time()}))
-    
+    print(f'Connected to client at {connection_addr}' +
+          f' on {datetime.datetime.now()}.')
+
+    await websocket.send(json.dumps({'type': 'connected', 'payload': 'true',
```

Comment:
https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas
If you try this does pycodestyle say you shouldn't have trailing commas?